protocol/epoch_test.go

```diff
@@ -0,0 +1,280 @@
+package protocol
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"fnd/blob"
+	"fnd/crypto"
+	"fnd/store"
+	"fnd/testutil/mockapp"
+	"fnd/util"
+
+	"github.com/stretchr/testify/require"
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+type epochTestSetup struct {
+	tp *mockapp.TestPeers
+	ls *mockapp.TestStorage
+	rs *mockapp.TestStorage
+}
+
+func TestEpoch(t *testing.T) {
+	name := "foobar"
+	tests := []struct {
+		name string
+		run  func(t *testing.T, setup *epochTestSetup)
+	}{
+		{
+			"syncs sectors when the local node has never seen the name before",
+			func(t *testing.T, setup *epochTestSetup) {
```

Comment:
These tests belong somewhere else, since they're not testing the very simple epoch method defined in `epoch.go`.

Reply:
Moved to `updater_test.go`